pages/integrations/inputs/okta_input.rst

```diff
@@ -0,0 +1,32 @@
+.. _okta_input:
+
+*********************
+Okta Log Events Input
+*********************
+
+.. attention:: This is an Enterprise Integrations feature and is only available since Graylog version 3.3 and thus requires an Enterprise license.
+
+
+This input allows Graylog to pull Okta Log Events from your okta domain.
+
+For this input plugin to function as expected, the following items need to be supplied in the input module.
+
+1) Domain name
+    The domain name associated with the IP address of the inbound event request (where the Okta Log Events are located).
+
+2) API Key
+    The API token that allows for polling Okta Log Events.
+
+3) Pull Log Event Since
```

Comment:
Need to add information about the Polling Interval.  Determines how often we will request new data from Okta.  Cannot be smaller than 5 seconds.